### DISCUSSION TITLE
Allow muting

### DISCUSSION THREAD

**Discussion by panoply** [community/community#13282] [Other Feature Feedback, Questions, & Ideas] [answered]
Some people I follow I don't care for their stars. It would be nice to mute certain folks activity. 

Thanks.

**Comment by sntwandy** [comment]
An option to stop receiving updates from repos, people, and other activities, has been requested by several users, apparently, the implementation of this action would be interesting.

**Reply by choony1** [reply]
☺️

**Comment by abhishekpatelmc** [comment]
There should be an option to stop following the conversation

**Comment by Layananbribanck** [comment]
>

**Comment by siminapasat** [comment, marked as answer]
This should be possible via filtering (remove the "stars" category") https://github.blog/changelog/2022-06-14-updates-to-github-feed-public-beta/

**Comment by Ammokhan1** [comment]
Excellent
